src/TrackingOptOut.php

```diff
@@ -107,14 +107,15 @@ public static function renderSettings() {
 
   public static function enqueueScriptsFirstHead() {
     wp_enqueue_script(Plugin::PREFIX . '-google_analytics_opt_out', Plugin::getBaseUrl() . '/dist/scripts/google-analytics-opt-out.js');
+    wp_localize_script(Plugin::PREFIX . '-google_analytics_opt_out', 'core_standards_opt_out_cookie_domain', '.' . ltrim(COOKIE_DOMAIN, '.'));
   }
 
   /**
    * @implements wp_enqueue_scripts.
    */
   public static function wp_enqueue_scripts() {
     wp_enqueue_script(Plugin::PREFIX . '-enable-opt_out', Plugin::getBaseUrl() . '/dist/scripts/enable-opt-out.js', ['jquery'], FALSE, TRUE);
-    wp_localize_script(Plugin::PREFIX . '-enable-opt_out', 'core_standards', ['opt_out_confirmation_message' => __('Google Analytics tracking has been disabled.', Plugin::L10N)]);
+    wp_localize_script(Plugin::PREFIX . '-enable-opt_out', 'core_standards_opt_out_confirmation_message', 'Google Analytics tracking has been disabled.', Plugin::L10N);
```

Comment:
```suggestion
    wp_localize_script(Plugin::PREFIX . '-enable-opt_out', 'core_standards_opt_out_confirmation_message', __('Google Analytics tracking has been disabled.', Plugin::L10N));
```

We lost the string translation here.